commit a2211c76c3b994099fd58a06d6072d7495d699cd
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Mar 18 18:18:35 2022 +0100

    scripts/dso-ordering-test.py: Fix C&P error in * callrefs processing
    
    The elf/dso-sort-tests-src subdirectory is not changed by this commit,
    so it seems that the cut-and-paste error was not material.
    
    Reviewed-by: Adhemerval Zanella  <adhemerval.zanella@linaro.org>

diff --git a/scripts/dso-ordering-test.py b/scripts/dso-ordering-test.py
index bde0406be9da14fc..ee476c810c76f1b0 100644
--- a/scripts/dso-ordering-test.py
+++ b/scripts/dso-ordering-test.py
@@ -551,17 +551,17 @@ def process_testcase(t):
         if obj in t.deps:
             deps = t.deps[obj]
             if '*' in deps:
-                t.deps[obj].remove('*')
+                deps.remove('*')
                 t.add_deps([obj], non_dep_tgt_objs)
         if obj in t.callrefs:
             deps = t.callrefs[obj]
             if '*' in deps:
-                t.deps[obj].remove('*')
+                deps.remove('*')
                 t.add_callrefs([obj], non_dep_tgt_objs)
     if "#" in t.deps:
         deps = t.deps["#"]
         if '*' in deps:
-            t.deps["#"].remove('*')
+            deps.remove('*')
             t.add_deps(["#"], non_dep_tgt_objs)
 
     # If no main program was specified in dependency description, make a
